.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ jobs:
 
     env:
       BASE_IMAGE_VERSION: "3.8-alpine3.11"
-      POETRY_VERSION: "1.0.8"
+      POETRY_VERSION: "1.1.4"
       TAG: "python3.8-poetry1.0"
```

Comment:
... und jetzt wird es mit den Tag ein bisschen blöd, oder? :( habe ich damals vorgeschlagen, poetry version nicht in dem Tag mitreinnehmen...

Spannend wird es auch jetzt mit Python 3.9. Wofür ist das image hier eigentlich nochmal benutzt?

Reply:
WIr haben damals ein neues Image erstellt `moneymeets-ci`, welches erstmal nur für das Pulumi Repository eingesetzt wurde. (https://moneymeets.atlassian.net/browse/MD-4448)
Der Plan war es eigentlich das neue `moneymeets-ci` Image durch dieses hier abzulösen, leider ist das Thema dann hinten runtergefallen und wurde nich angegangen. :(

Ich werde dafür ein Ticket erstellen, sodass die Backend Repos auf das andere Image umgestellt werden, dort gibt es dann auch nur ein Tag `latest` und Tags für die Feature Branches zum Testen.

Ich würde gerne dieses Image hier updaten, da @felix11h ein Problem festgestellt hat, was in den Pipelines nicht aufgefallen ist, aufgrund der alten Poetry Version... :(

Reply:
https://moneymeets.atlassian.net/browse/MD-4739

Reply:
Also dieses image ist noch für alle Backends benutzt?! Ich frage mich wieso denn es überhaupt jetzt noch funktioniert mit CI und fixe Version von Python - nur Zufall?

Reply:
Das ist mir auch

> Also dieses image ist noch für alle Backends benutzt?! Ich frage mich wieso denn es überhaupt jetzt noch funktioniert mit CI und fixe Version von Python - nur Zufall?

Das ist mir auch ein Rätsel. In den Actions nutzen wir definitv dieses Image mit Poetry 1.0.8...